suite-native/connection-status/src/useIsOfflineBannerVisible.tsx

```diff
@@ -6,9 +6,11 @@ import { selectIsOnboardingFinished } from '@suite-native/settings';
 
 export const useIsOfflineBannerVisible = () => {
     const isOnboardingFinished = useSelector(selectIsOnboardingFinished);
-    const { isInternetReachable } = useNetInfo();
+    const { isConnected } = useNetInfo({
+        reachabilityUrl: 'https://cdn.trezor.io/204',
+    });
 
-    const isReachable = isInternetReachable ?? true;
+    const isReachable = isConnected ?? true;
 
     return !isReachable && isOnboardingFinished;
```

Comment:
I liked the previous version a bit more:

```
    return !isConnected && isOnboardingFinished;
```

Reply:
c2d02a7f69